include/seqan3/search/search.hpp

```diff
@@ -197,7 +209,20 @@ inline auto search(queries_t && queries,
 }
 
 //!\cond DEV
-//! \overload
+// Overload for a single query (not a collection of queries)
+//!\overload
+template <fm_index_specialisation index_t,
+          typename query_t,
+          typename configuration_t = decltype(search_cfg::default_configuration)>
+    requires !std::ranges::forward_range<std::ranges::range_value_t<query_t>>
+inline auto search(query_t && query,
+                   index_t const & index,
+                   configuration_t const & cfg = search_cfg::default_configuration)
+{
+    return search(std::views::single(std::forward<query_t>(query)), index, cfg);
```

Comment:
```suggestion
    return search(std::views::single(std::move(query)), index, cfg);
```

Reply:
views::single always takes per copy so we can take per value and always move here.